comp/checks/windowseventlog/windowseventlogimpl/check/ddlog_submitter.go

```diff
@@ -101,7 +100,15 @@ func (s *ddLogSubmitter) enrichEvent(m *windowsevent.Map, e *eventWithMessage) e
 	}
 	defer evtapi.EvtClosePublisherMetadata(e.evtapi, pm)
 
-	windowsevent.AddRenderedInfoToMap(m, e.evtapi, pm, e.winevent.EventRecordHandle)
+	eh := e.winevent.EventRecordHandle
+	task, _ := e.evtapi.EvtFormatMessage(pm, eh, 0, nil, evtapi.EvtFormatMessageTask)
+	opcode, _ := e.evtapi.EvtFormatMessage(pm, eh, 0, nil, evtapi.EvtFormatMessageOpcode)
+	level, _ := e.evtapi.EvtFormatMessage(pm, eh, 0, nil, evtapi.EvtFormatMessageLevel)
+
+	_ = m.SetMessage(e.renderedMessage)
+	_ = m.SetTask(task)
+	_ = m.SetOpcode(opcode)
+	_ = m.SetLevel(level)
```

Comment:
rendering was handled by `windowsevent.AddRenderedInfoToMap` but I expanded the function since we already have the message, and rendering the message is expensive

Reply:
ah, makes total sense